Sources/BraintreeCoreSwift/BTAPIClientSwift.swift

```diff
@@ -1,5 +1,441 @@
+                    }
+                }
+            }
+
+            completion(configuration, nil)
+        }
+    }
+
+    /// Fetches a customer's vaulted payment method nonces.
+    /// Must be using client token with a customer ID specified.
+    ///  - Parameter completion: Callback that returns either an array of payment method nonces or an error
+    public func fetchPaymentMethodNonces(completion: @escaping ([BTPaymentMethodNonce]?, Error?) -> Void) {
+        fetchPaymentMethodNonces(false, completion: completion)
+    }
+
+    /// Fetches a customer's vaulted payment method nonces.
+    /// Must be using client token with a customer ID specified.
+    ///  - Parameters:
+    ///   - defaultFirst: Specifies whether to sort the fetched payment method nonces with the default payment method or the most recently used payment method first
+    ///   - completion: Callback that returns either an array of payment method nonces or an error
+    public func fetchPaymentMethodNonces(_ defaultFirst: Bool, completion: @escaping ([BTPaymentMethodNonce]?, Error?) -> Void) {
+        if clientToken == nil {
+            completion(nil, BTAPIClientError.notAuthorized)
+            return
+        }
+
+        let defaultFirstValue: String = defaultFirst ? "true" : "false"
+        let parameters = [
+            "default_first": defaultFirstValue,
+            "session_id": metadata?.sessionID
+        ]
+
+        get("v1/payment_methods", parameters: parameters as NSDictionary) { body, response, error in
+            if let error {
+                completion(nil, error)
+                return
+            }
+
+            var paymentMethodNonces: [BTPaymentMethodNonce]? = []
+
+            body?["paymentMethods"].asArray()?.forEach { paymentInfo in
+                let type: String? = paymentInfo["type"].asString()
+                let paymentMethodNonce: BTPaymentMethodNonce? = BTPaymentMethodNonceParser.shared.parseJSON(paymentInfo, withParsingBlockForType: type)
+
+                if let paymentMethodNonce {
+                    paymentMethodNonces?.append(paymentMethodNonce)
+                }
+            }
+
+            completion(paymentMethodNonces, nil)
+        }
+    }
+
+    /// :nodoc:
+    /// Perfom an HTTP GET on a URL composed of the configured from environment and the given path.
+    /// - Parameters:
+    ///   - path: The endpoint URI path.
+    ///   - parameters: Optional set of query parameters to be encoded with the request.
+    ///   - completion:  A block object to be executed when the request finishes.
+    ///   On success, `body` and `response` will contain the JSON body response and the
+    ///   HTTP response and `error` will be `nil`; on failure, `body` and `response` will be
+    ///   `nil` and `error` will contain the error that occurred.
+    @objc(GET:parameters:completion:)
+    public func get(_ path: String, parameters: NSDictionary? = nil, completion: @escaping RequestCompletion) {
```

Comment:
We can now! We were waiting until BTAPIClient was converted to update since it uses the HTTP methods under the hood. Pushing up this change and removing the TODO we had for it. 🚀